readme.md

```diff
@@ -13,45 +13,46 @@
+| [SurveyMonkey](https://www.surveymonkey.com/) | SurveyMonkey is an online survey software that helps you to create, edit, and run professional online surveys. The basic plan allows you to have only 10 questions per survey with a maximum of 100 responses per survey (you can have unlimited questions and responses per survey in the premium plans). |
+| [Airtable](https://www.https://airtable.com/) | Airtable is a platform that makes it easy to build powerful, custom applications. It can also be used to track job interviews, manage large-scale video production, and even collect data. (this tool is a freemium). |
 
 ## Design
 
->Websites and resources for making designs, mockups and flyers.
+>Websites and resources for making designs, mockups, and flyers.
 
 | Website&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; | Description |
 | -------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------ |
 | [Canva](https://www.canva.com/) | Canva is a graphic design platform that is used to create social media graphics and presentations.|
 
 ## Documentation
 
->Websites and resources for making documentation, making notes, planning and staying organized.
+>Websites and resources for making documentation, making notes, planning, and staying organized.
 
 | Website&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; | Description |
 | -------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------ |
-| [Notion](https://www.notion.so/) | Notion is a freemium productivity and note-taking web application developed by Notion Labs Inc. It is offering many organizational tools including task management, project tracking, to-do lists, bookmarking, and more.|
-| [Airtable](https://www.https://airtable.com/) | Airtable is a platform that makes it easy to build powerful, custom applications. It can also be used to do everything from tracking job interviews to managing large-scale video production and even data collection. (this tool is a freemium). |
+| [Notion](https://www.notion.so/) | Notion is a freemium productivity and note-taking web application developed by Notion Labs Inc. It offers many organizational tools, including task management, project tracking, to-do lists, bookmarking, and more.|
+| [Airtable](https://www.https://airtable.com/) | Airtable is a platform that makes it easy to build powerful, custom applications. It can also be used to track job interviews, manage large-scale video production, and even collect data. (this tool is a freemium). |
```

Comment:
```suggestion
| [Airtable](https://www.https://airtable.com/) | Airtable is a low-code platform that makes it easy to build powerful, custom applications. It can also be used for other things such as tracking job interviews, managing large-scale video production, and even collecting data. (this tool is a freemium). |
```